src/libraries/System.ComponentModel.TypeConverter/src/System/ComponentModel/ReflectTypeDescriptionProvider.cs

```diff
@@ -1328,11 +1365,27 @@ private static object SearchIntrinsicTable(Hashtable table, Type callingType)
 
                 if (type != null)
                 {
-                    hashEntry = CreateInstance(type, callingType);
                     if (type.GetConstructor(s_typeConstructor) == null)
                     {
+                        bool success = ConverterConstructorFuncs.TryGetValue(type, out Func<object> ctorFunc);
```

Comment:
Is `CreateInstance(Type, Type)` still used?

Reply:
Yes, there are 4 remaining references.